lib/intercom-rails/script_tag.rb

```diff
@@ -26,6 +27,19 @@ def initialize(options = {})
       self.nonce = options[:nonce]
     end
 
+    def remove_user_cookie_on_logout
+      if (find_current_user_details == {} && controller.response.request.cookies["intercom-session-#{IntercomRails.config.app_id}"])
```

Comment:
self.user_details returns { app_id: "current_app_id"} 
I could check if self.user_details ==  { app_id: "current_app_id" }  but I thought it was more readable to check if find_current_user_details == {} even if it's not as efficient... What do you think ?